docs/AddingDoppl.md

```diff
@@ -557,9 +557,159 @@ show the test results in the console, and you should see that all tests pass:
 
-## Step #14: Write the UI
+## Step #14: Access the Generated Code From Swift
 
-TBD
+Most likely, your job is to ship an app with a user interface, not just
+a working test suite. For that, you will need to be able to access the
+generated code from your Swift files in `iosApp`, so that you can obtain
+the model data loaded from the network. From there, you can craft a UI to
+display that data.
+
+Here, we will focus on just getting you access to the model data via the
+view-model. Creating a suitable iOS UI that uses this data is left as an exercise
+for the reader.
+
+### `AppDelegate`
+
+In the `iosApp` project, click on `AppDelegate.swift` and
+
+- Add `import doppllib` to the list of imports
+- Add `DopplRuntime.start()` to `application()`
+
+This should give you something like this:
+
+```swift
+import UIKit
+import doppllib
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
+        DopplRuntime.start()
+        return true
+    }
+
+    func applicationWillResignActive(_ application: UIApplication) {
+        // Sent when the application is about to move from active to inactive state. This can occur for certain types of temporary interruptions (such as an incoming phone call or SMS message) or when the user quits the application and it begins the transition to the background state.
+        // Use this method to pause ongoing tasks, disable timers, and invalidate graphics rendering callbacks. Games should use this method to pause the game.
+    }
+
+    func applicationDidEnterBackground(_ application: UIApplication) {
+        // Use this method to release shared resources, save user data, invalidate timers, and store enough application state information to restore your application to its current state in case it is terminated later.
+        // If your application supports background execution, this method is called instead of applicationWillTerminate: when the user quits.
+    }
+
+    func applicationWillEnterForeground(_ application: UIApplication) {
+        // Called as part of the transition from the background to the active state; here you can undo many of the changes made on entering the background.
+    }
+
+    func applicationDidBecomeActive(_ application: UIApplication) {
+        // Restart any tasks that were paused (or not yet started) while the application was inactive. If the application was previously in the background, optionally refresh the user interface.
+    }
+
+    func applicationWillTerminate(_ application: UIApplication) {
+        // Called when the application is about to terminate. Save data if appropriate. See also applicationDidEnterBackground:.
+    }
+
+
+}
+```
+
+This is the same as what we did in the `iosTest` project.
```

Comment:
Minor nitpick: I believe iosTest uses `testdoppllib`.

Reply:
Oops, yes -- fixed in fresh commit. Thanks!